projects/amun/index.js

```diff
@@ -1,10 +1,32 @@
 const sdk = require('@defillama/sdk');
-const abi = require('./abi.json')
+const abi = require('./abi.json');
+const axios = require('axios');
+const solana = require('../helper/solana');
 
+const PECO_ADDRESS = '0xA9536B9c75A9E0faE3B56a96AC8EdF76AbC91978';
 const DFI_ADDRESS = '0xA9536B9c75A9E0faE3B56a96AC8EdF76AbC91978';
-const DMX_ADDRESS = '0x1660F10B4D610cF482194356eCe8eFD65B15bA83';
+const SOLI_ADDRESS = '8JnNWJ46yfdq8sKgT1Lk4G7VWkAA8Rhh7LhqgJ6WY41G';
+
+async function getSolanaTokenRate(address) {
+    return await axios.get("https://lima-api.amun.com/tokenRate/APygy51vntbQQsYmgkF1CeMsAn5Tv1rsVUB6nee3wJpY?chain=solana&address="+address);
```

Comment:
@Grzesiek-P , don't we have any way to query past NAV of SOLI?

Reply:
solana doesn't support timestamping as each block is of different timespan - a known issue for solana